pyproject.toml

```diff
@@ -70,3 +70,44 @@ disable = [
     "no-name-in-module",
     "useless-return",
 ]
+
+[tool.ruff]
+target-version = "py37"
```

Comment:
Why only set the lowest supported Python version here?

Reply:
This is so ruff doesn’t auto fix with styles Python 3.7 doesn’t support.